### DISCUSSION TITLE
Is it possible to embed Wireguard?

### DISCUSSION THREAD

**Discussion by q962** [SideStore/SideStore#628] [Ideas]
Does ios allow the app to automatically connect to a VPN when it is opened in the background?
If it works, the sidestore is fully automated.

Are there some issues that make it impossible?
For example, one should wait for Wireguard to provide a trigger mechanism instead of letting developers do it

**Comment by f1shy-dev** [comment]
Not really since making a VPN app on iOS requires paid entitlements, which would defeat the point of SideStore in the first place
